hail/www/style.css

```diff
@@ -1,14 +1,10 @@
 /* https://gist.githubusercontent.com/killercup/5917178/raw/40840de5352083adb2693dc742e9f75dbb18650f/pandoc.css */
 
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap'); */
+
 html {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

Comment:
This looks good on the native system, and removes the need to load a font. Fonts need to load before the css that uses them is rendered, so that represents a load-blocking step (and the fonts are often large). This font-family covers Mac (and Safari),  Chromium, Android, and Windows native fonts.